torch/onnx/symbolic_helper.py

```diff
@@ -738,6 +738,13 @@ def _optional_input_placeholder_tensor(g):
     n.setType(OptionalType.ofTensor())
     return n
 
+def _handle_reduce_dim_none(g, self, op_name):
+    dim_size = _get_tensor_dim_size(self.node().output(), 0)
```

Comment:
```suggestion
    dim_size = _get_tensor_dim_size(self, 0)
```
`self.node().output()` == `self` if and only if `self.node()` has a single output.